test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -304,51 +313,6 @@ TEST_F(BlockQueryTest, GetBlocksFrom1) {
  * @when read block #1
  * @then get no blocks
  */
-TEST_F(BlockQueryTest, GetBlockButItIsNotJSON) {
```

Comment:
Why these tests was removed?

Reply:
Ops, removed a bit more that was intended